### PR TITLE
data acquisition now gets pool info and pool config

### PR DESCRIPTION
**Pull request by dpaiton**

### CODE REVIEW

**Review by slundqui — APPROVED** [review]
LGTM

File: data/acquire_transactions.py [hunk 1]
```diff
@@ -31,33 +31,32 @@ def default(self, o):
...
+    if isinstance(obj, dict):
+        return {key: recursive_dict_conversion(value) for key, value in obj.items()}
     if hasattr(obj, "items"):
         return {key: recursive_dict_conversion(value) for key, value in obj.items()}
     return obj
 
 
 def get_event_object(web3_instance, contract, log):
-    """TODO"""
+    """Retrieves the event object and anonymous types fora  given contract and log"""
```
Comment (wakamex) [inline data/acquire_transactions.py:59]: typo

File: elfpy/utils/apeworx_integrations.py [hunk 2]
```diff
@@ -21,13 +22,16 @@
 from ape.types import AddressType, ContractType
 from ape_accounts.accounts import KeyfileAccount
 
-from elfpy import MAXIMUM_BALANCE_MISMATCH_IN_WEI, SECONDS_IN_YEAR, WEI, simulators, time, types
+from elfpy import (MAXIMUM_BALANCE_MISMATCH_IN_WEI, SECONDS_IN_YEAR, WEI,
```
Comment (wakamex) [inline elfpy/utils/apeworx_integrations.py:25]: unnecessary tuple brackets

File: elfpy/utils/apeworx_integrations.py [hunk 3]
```diff
@@ -21,13 +22,16 @@
...
+                   simulators, time, types)
+from elfpy.markets.hyperdrive import (AssetIdPrefix, HyperdriveMarketState,
```
Comment (wakamex) [inline elfpy/utils/apeworx_integrations.py:27]: unnecesssary tuple brackets

File: elfpy/utils/apeworx_integrations.py [hunk 4]
```diff
@@ -593,14 +597,14 @@ def get_pool_state(txn_receipt: ReceiptAPI, hyperdrive_contract: ContractInstanc
...
-def _snake_to_camel(_snake):
+def snake_to_camel(_snake):
     """Convert snake_case to camelCase."""
     return "".join(word.capitalize() for word in _snake.split("_"))
 
 
-def _camel_to_snake(_camel):
-    """Convert camelCase to snake_case."""
-    return _camel.lower().replace(" ", "_")
+def camel_to_snake(camel_string: str) -> str:
```
Comment (wakamex) [inline elfpy/utils/apeworx_integrations.py:605]: this should be a separate PR, I already have one up for this:
https://github.com/delvtech/elf-simulations/pull/537

**Review by wakamex — APPROVED** [review]
lgtm

File: elfpy/utils/apeworx_integrations.py (hunk 3 above)
Comment (dpaiton) [reply elfpy/utils/apeworx_integrations.py:27]: black did this, I think it is necessary bc the line is too long and you need brackets for multi-line imports
